tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
+
+## Maven installation
+
+Maven is probably the most popular build tool in Java world these days. It's available in official Fedora repositories, in form of RPM. To install it, simply type:
+
+```
+# dnf install maven
+```
```

Comment:
We have libraries from Maven Central Repository in Fedora, but they are not used by developers (or they are used, but very very rarely). Trying to explain here how the packaged artifacts should be used during development could only confuse developers (the process is not straightforward).